doc/cephadm: move services under doc/cephadm/services ? (#43150)
ceph/ceph: doc/cephadm/services/index.rst

Service Management
==================

> A service is a group of daemons configured together. See these chapters
> for details of how do deploy individual services:

Reviewer: ```suggestion
A service is a group of daemons configured together. See these chapters
for details on individual services:
```
I feel like some of these sections cover a lot more than just deploying the services and this sounds a bit nicer.
